### ISSUE TITLE
Black lines on every textures

### ISSUE THREAD

**Issue by TuchoChort** [alphazolam/fmt_nd_pak#2]
Hello. I'm trying to export the models from TLOU P1 (PC version), but everytime I open any model, the textures are low res and has black lines on it. I did the 3rd step, but it didn't help.

**Comment by bazq3D** [comment]
you need to open a folder for every .psarc file,than dump them to their folder one by one. this fixed my issue.

**Comment by JanOutThere** [comment]
I have still the same  the same Problem!!!

**Comment by JanOutThere** [comment]
"you need to open a folder for every .psarc file,than dump them to their folder one by one. this fixed my issue." what do you mean exactly with that?

**Comment by sesang82** [comment]
me too.... how to solve it

**Comment by amazingoomoo** [comment]
Yes I have this same issue too, when I export the model the textures are tiny, are stretched vertically and have thick black lines across them. What does "you need to open a folder for every .psarc file,than dump them to their folder one by one." mean?

**Comment by hhjmirror** [comment]
> 你好。我正在尝试从 TLOU P1（PC 版）导出模型，但每次打开任何模型时，纹理都是低分辨率的，上面有黑线。我做了第三步，但没有帮助。

请问第三条可选内容的编辑指向纹理文件夹的操作如何完成？

**Comment by EOMNS** [comment]
up ! :( same problem here

**Comment by KayhoticGames** [comment]
2 years later and this problem still exists with zero CLEAR help on what to do...
